DenyHosts/regex.py

```diff
@@ -25,7 +25,7 @@
 
 FAILED_ENTRY_REGEX8 = re.compile(r"""authentication error for (?P<user>.*) .*from (?P<host>.*)""")
 
-FAILED_ENTRY_REGEX9 =  re.compile(r"""Connection closed by (?P<host>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) \[preauth\]""")
+FAILED_ENTRY_REGEX9 =  re.compile(r"""Connection closed by (?P<host>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})( port \d+) \[preauth\]""")
```

Comment:
I'd prefer if the port would be ```( port \d+)?```.  That way for older operating systems that don't have the port defined, it won't break them.

Reply:
@xobtoor can you update your code with the requested changes above?

Reply:
@josev814 done